harvard-huit/Harvard-Geospatial-Library
# Robots vim

*Remove vim tiny, edit robots.txt, fix bug for index maps**
* * *

# What does this Pull Request do?
This PR implements 3 small changes to allow for a new release next week

1) Edit robots.txt to only allow google (per Jason)
2) Remove vim.tiny from container image (left in DockerfileLocal)
3) Fix a bug with certain accented a's not being converted to links (see: https://hgl.harvard.edu/catalog/harvard-g7630-s100-a3-index-gr -- the request scan button is not right)

# How should this be tested?
The bug fix can be tested by bringing this branch up locally, and visiting the index map record (already pre-indexed in dev). 

The other 2 things cannot be tested locally. However, I have deployed this image to dev, and so you can go to dev, shell into the container, and see that vim isn't there anymore (type vim, hit enter, get told no vim). You can also see the button fix in dev.

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? N
- integration tests? N

# Interested parties
@enriquediaz
